packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -14,16 +14,16 @@ Customize error pages as part of the sign-in flow.
 
 **What you need**
 
-* An [Okta Developer Edition organization](/signup)
-* A [customized Okta URL domain](/docs/guides/custom-url-domain/), unless you are using the [Brands API](#use-the-brands-api)
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* [Customized Okta URL domain](/docs/guides/custom-url-domain/), unless you are using the [Brands API](#use-the-brands-api)
 
 **Sample code**
 
 [Customization examples](#customization-examples)
 
 ---
 
-## Overview
+## About error page customization
```

Comment:
"Error page customization overview"? The "About" reads a bit strangely, although maybe it's just me.

Reply:
👍